components/create/createLoader.tsx

```diff
@@ -16,6 +18,11 @@ function Success() {
         (async () => {
             if (loading && address.status == 0) {
                 try {
+                    const metadataMap = new MichelsonMap();
+                    metadataMap.set(
```

Comment:
Can you put `metadata.json` instead?
metadata.json can be found here.
https://github.com/marigold-dev/multisig/releases/tag/0.0.6

FYI: https://github.com/ligolang/multisig-cameligo/blob/main/deploy/deploy.ts#L78-L80

Reply:
fixed